sdk/turing/router/config/router_config.py

```diff
@@ -211,6 +211,10 @@ def experiment_engine(self, experiment_engine: Union[ExperimentConfig, Dict]):
             self._experiment_engine = experiment_engine
         elif isinstance(experiment_engine, dict):
             self._experiment_engine = ExperimentConfig(**experiment_engine)
+            if self._experiment_engine.config is not None and \
```

Comment:
some comment would help, this is assuming the backend knows to reuse old passkey when this unset and this seems unintuitive to begin with 😓

Reply:
Ah that's true; got it, I've added the following comments:
```
# This block sets the passkey (encrypted when retrieved from the Turing API server) to None to prevent users
# from sending a router config with an already encrypted passkey back to API server when updating a router.
#
# When the passkey value is not set, the Turing API server is able to automatically retrieve the correct
# passkey from the existing router version (assuming it has been configured with the same standard
# experiment engine and the same client username).
```